bincrafters_conventions/actions/update_t_linux_image.py

```diff
@@ -0,0 +1,14 @@
+def update_t_linux_image(main, file):
+    """ Update the Travis Linux base image (i.e. native image, not the images we run via Docker)
+    """
+
+    image_update_mapping = {
+        "sudo: required": "dist: xenial"
+    }
+
+    for old_image_version, new_image_version in image_update_mapping.items():
+        if main.file_contains(file, old_image_version):
```

Comment:
```suggestion
        if main.file_contains(file, old_image_version) and \
           not main.file_contains(file, new_image_version):
```

Reply:
I didn't think about this because `.travis.yml` files should have never had both keys in the first place and if I recall correctly our templates also never suggested that

That said, I think it doesn't hurt to add this check 🤔

Reply:
Both keys: `dist: xenial` and `sudo: required`